### PR TITLE
Upgrade to Saleor 3.5.x and introduce Make commands

### PR DESCRIPTION
**Pull request by lkostrowski**
This PR bumps submodules to 3.5.x versions.

Introduces Makefile for one-command scripts

Also, introduce some docs upgrades

Related to #157

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,8 +169,6 @@ docker-compose down --volumes db
 If you have any questions or feedback, do not hesitate to contact us via GitHub or Gitter:
 
 - https://github.com/saleor/saleor/discussions
-- https://gitter.im/mirumee/saleor
```
Comment (magul) [inline README.md:151]: why do we remove that? I thought it's still supported, I saw @karolkielecki comments there no more than month old.

Comment (lkostrowski) [reply README.md:151]: Due to SEO, we are moving to Github discussions

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+bootstrap:
```
Comment (krzysztofwolski) [inline Makefile:1]: Wonder if adding "cleanup" command would be a good idea? It would tear down the db and its volume.

**Review by krzysztofwolski — APPROVED** [review]
Might be a good idea 👍

**Review by peelar — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (lkostrowski) [reply Makefile:1]: As discussed, @krzysztofwolski will open separate issue for it

Comment (krzysztofwolski) [reply Makefile:1]: #159 task added 👍

**Pull request merged**